### ISSUE TITLE
Add ability to automatically copy attachments to a directory with the map

### ISSUE THREAD

**Issue by p-himik** [freeplane/freeplane#103]
Right now, if I attach an image to some node, Freeplane won't copy the image anywhere. The issue here is that I cannot share and backup the map by itself in a simple way - I have to first copy something that I want to attach to the directory with the map, and only then attach it.

**Comment by TaissyrSt** [comment]
Hi,
Do you suggest adding the attachment viewer feature that allows you to view the images as feature attachments?

**Comment by p-himik** [comment]
No, I didn't say anything about viewing attachments. The point is to just make a map with external attachments portable.
